src/app/components/repeater/index.html

```diff
@@ -26,6 +26,11 @@
       </li>
     </ul>
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="activeIndex"
```

Comment:
This should be the first item in the table because it comes first alphabetically. ... I'll pull down the branch and fix this, but adding a comment in case you are still working on this.